build/docker/services/base/container-install-htcondor.sh

```diff
@@ -102,12 +102,16 @@ elif [[ $OS_ID == ubuntu ]]; then
     fi
     $apt_install gnupg2 wget
 
-    # TODO Is there a per-series key I should be using instead?
     set -o pipefail
-    wget -qO - "${repo_base_ubuntu}/HTCondor-Release.gpg.key" | apt-key add -
+    if [[ $HTCONDOR_SERIES = 8.9 ]]; then
+        wget -qO - "https://research.cs.wisc.edu/htcondor/ubuntu/HTCondor-Release.gpg.key" | apt-key add -
+    else
+        wget -qO - "https://research.cs.wisc.edu/htcondor/repo/keys/HTCondor-$SERIES-Key" | apt-key add -
```

Comment:
```suggestion
        wget -qO - "https://research.cs.wisc.edu/htcondor/repo/keys/HTCondor-${HTCONDOR_SERIES}-Key" | apt-key add -
```

Reply:
You're right. I made the change, though I wonder why it built successfully with the broken variable...